docs/user/guides/setup/monorepo.rst

```diff
@@ -0,0 +1,92 @@
+How to use a .readthedocs.yaml file in a sub-folder
+===================================================
+
+This guide shows how to configure a Read the Docs project to use a custom path for the ``.readthedocs.yaml`` build configuration.
+*Monorepos* that have multiple documentation projects in the same Git repository can benefit from this feature.
```

Comment:
```suggestion
:term:`Monorepos` that have multiple documentation projects in the same Git repository can benefit from this feature.
```

I'm not sure if we have it defined, tho 😄

Reply:
Ah yeah, I would add a glossary item if I could think of something special from RTD's usage of the term. But will just start by linking https://en.wikipedia.org/wiki/Monorepo since it's a pretty good simple article.